src/internal/remote/toBKTException.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiServerException, BadRequestException, BKTException, FeatureNotFoundException, InvalidHttpMethodException, UnauthorizedException, UnknownException } from '../../BKTExceptions'
+import { ErrorResponse } from '../model/response/ErrorResponse'
+import { FetchResponseLike } from './fetch'
+
+export const toBKTException = async (response: FetchResponseLike): Promise<BKTException> => {
+  const errorBody: ErrorResponse | null = await response.json()
+  const error = errorBody?.error
+
+  switch (response.status) {
```

Comment:
The server errors have changed to the following.
https://github.com/bucketeer-io/android-client-sdk/issues/56

Let me know if you prefer to do this in another PR.

Reply:
got it. updated in [`801b304` (#13)](https://github.com/bucketeer-io/javascript-client-sdk/pull/13/commits/801b304e9fd9f8649b874a473149be55e78f1b39)